webots_ros2_epuck/resource/ros2_control.yml

```diff
@@ -16,5 +16,4 @@ diffdrive_controller:
     wheel_separation: 0.05685
     wheel_radius: 0.02
 
-    publish_rate: 50.0
```

Comment:
Do we simply remove it as it has no effect and I will put it again when my PR on `ros2control` is accepted, or should we keep it anyway?

Reply:
I would rather put it later.

Reply:
Fine for me then.